android/java/v2/AmpliApp/app/src/main/java/com/example/ampliapp/EnrichmentPlugin.java

```diff
@@ -0,0 +1,45 @@
+package com.example.ampliapp;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.amplitude.core.Amplitude;
+import com.amplitude.core.events.BaseEvent;
+import com.amplitude.core.platform.Plugin;
+
+import java.util.HashMap;
+
```

Comment:
Could you please add a description of what this plugin does to help customers navigate the example.